httpcache_test.go

```diff
@@ -1569,6 +1580,77 @@ func TestTransportVaryMatchesWithMask(t *testing.T) {
 			t.Fatalf(`XFromCache header isn't "1": %v`, resp.Header.Get(XFromCache))
 		}
 	}
+
+	// Verify initial request also still caches
+	req.Header.Set("Accept", "text/plain")
+	{
+		resp, err := s.client.Do(req)
+		if err != nil {
+			t.Fatal(err)
+		}
+		defer resp.Body.Close()
+		if resp.Header.Get(XFromCache) != "1" {
+			t.Fatalf(`XFromCache header isn't "1": %v`, resp.Header.Get(XFromCache))
+		}
+	}
+}
+
+func TestLastModifiedWithVaryIgnoreMask(t *testing.T) {
+	// Initial request
+	resetTest()
```

Comment:
it doesn't look like you're setting the mask on the transport anywhere. if it's still set because of the last test can you clear it on the last test and set it here?

Reply:
ah whoops you are correct. done.